src/libDirectoryService/FinalBlockPostProcessing.cpp

```diff
@@ -567,6 +567,23 @@ bool DirectoryService::ProcessFinalBlockConsensusCore(
     return false;
   }
 
+#ifdef VC_TEST_FB_SUSPEND_RESPONSE
+  ConsensusCommon::State state = m_consensusObject->GetState();
+
+  static const int afterVcCount = 0;
+  if (state == CHALLENGE_DONE && m_mode == PRIMARY_DS)
+    if (m_viewChangeCounter == 0  // Initial ds leader
```

Comment:
I think `m_viewChangeCounter == 0` will be enough. It will stop consensus, view change, new leader restart consensus and move on. Then next epoch repeat

Reply:
To make the test more efficient, maybe don't halt it at vaccous epoch